INSTALL.md

```diff
@@ -156,7 +156,7 @@ Installing prerequisites on CentOS 8 or Fedora>26:
 ```sh
 sudo dnf install libtiff-devel libjpeg-devel libzip-devel freetype-devel \
                  libwebp-devel tk-devel redhat-rpm-config \
-                 python3-virtualenv libffi-devel openssl-devel
+                 python3-virtualenv libffi-devel openssl-devel python3-devel phycopg2 
```

Comment:
Thank you for the contribution! Could you please double-check these package names? I thought the Fedora package for psycopg2 was named `python3-psycopg2`

Reply:
right, my bad, yes it is is `python3-psycopg2`, and I'm uncertain if `libpq-devel` would be better